src/index.ts

```diff
@@ -0,0 +1,17 @@
+export class Challenge {
+    num : number;
+    constructor(){
```

Comment:
Empty constructors are the default and can be avoided.
Also try to always mark it with its visibility just for better clarity (`public` is the default level and is good in most cases)